runtime/j9vm/java11vmi.c

```diff
@@ -1455,8 +1455,32 @@ JVM_ConstantPoolGetClassRefIndexAt(JNIEnv *env, jobject arg1, jlong arg2, jint a
 
 jobjectArray JNICALL
 JVM_GetVmArguments(JNIEnv *env) {
-	assert(!"JVM_GetVmArguments unimplemented"); /* Jazz 108925: Revive J9JCL raw pConfig build */
-    return NULL;
+	J9VMThread* currentThread = (J9VMThread*)env;
+	J9JavaVM* vm = currentThread->javaVM;
+	J9InternalVMFunctions* internalFunctions = vm->internalVMFunctions;
+	jclass vmJniClass = NULL;
+	jobjectArray result = NULL;
+	J9Class* vmClass = J9VMCOMIBMOTIVMVM_OR_NULL(vm);
+
+	if (NULL == vmClass) {
+		internalFunctions->setCurrentException(currentThread, J9VMCONSTANTPOOL_JAVALANGINTERNALERROR, NULL);
+	} else {
+		jclass vmJniClass = NULL;
+		jmethodID mid = NULL;
+
+		vmJniClass = (jclass)internalFunctions->j9jni_createLocalRef(env, vmClass->classObject);
+
+		mid = (*env)->GetStaticMethodID(env, vmJniClass, "getVMArgs", "()[Ljava/lang/String;");
```

Comment:
There's a bug here - the vm internal functions have to be used under vm access while the JNI `GetStaticMethodID` can't.  We're missing some vm access related calls in this PR.

Reply:
I will open a new pr to fix that

Reply:
pr here: https://github.com/eclipse/openj9/pull/3023